tools/calyx-writer/lib.rs

```diff
@@ -353,15 +362,22 @@ impl CalyxWriter for Cell {
 /// A guard for an [`Assignment`].
 #[derive(PartialEq, Eq, Debug)]
 pub enum Guard {
-    None,
-    Constant(u64),
+    True,
     Port(Port),
     And(Box<Guard>, Box<Guard>),
```

Comment:
To state the obvious, which I'm sure you'll add later. This is missing Or, Not, and BinOps. Also worth noting that `Port`'s are only allowed as individual guard elements when they are 1bit values, otherwise they have to be part of some comparison operator